common/src/main/java/com/revenuecat/purchases/common/verification/Signature.kt

```diff
@@ -0,0 +1,83 @@
+package com.revenuecat.purchases.common.verification
+
+import android.util.Base64
+import androidx.annotation.VisibleForTesting
+
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal enum class SignatureComponent(val size: Int) {
```

Comment:
Is it possible in Kotlin to make this a nested type of `Signature`? So this becomes `Signature.Component`?

Reply:
Yeah we can do that 👍

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1109/commits/c7d4619a3bbff534a894c1f0080e80cadf3d418d

Reply:
Awesome.